keras/engine/training_test.py

```diff
@@ -1427,6 +1428,30 @@ def test_outputs_are_floats(self):
     self.assertIsInstance(loss, float)
     self.assertIsInstance(accuracy, float)
 
+  @keras_parameterized.run_all_keras_modes(always_skip_v1=True)
+  @unittest.expectedFailure
```

Comment:
Is this line expected here?

Reply:
It is expected, an expected failing test will need to fail only when it will be solved.